<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseOptions</class>
 <widget class="QMainWindow" name="DatabaseOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Options</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnBack">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="wizardPanel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="lblInitialPrompt">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>381</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>This wizard allows you to connect to </string>
      </property>
     </widget>
     <widget class="QLabel" name="lblInitialPrompt_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>381</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>the Abaci Database Server.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>201</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resources.qrc">:/DB/resources/DB.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLineEdit" name="txtHostname">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hostname / IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Connection </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtUsername">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPassword">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkShowPassword">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>170</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="numPort">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>80</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
      </property>
      <property name="value">
       <number>1433</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTestConnection">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Test Connection</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lstTestConnection">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>441</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Database</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDatabase">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Table</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbClientTable">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblClientTable">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Client Table</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblJobTable">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Results Table</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbResultTable">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRefreshTables">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh Tables</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVerifyTables">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Verify Tables</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCreateClients">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Clients</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCreateResults">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Results</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDatabaseless">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Databaseless</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txtHostname</tabstop>
  <tabstop>numPort</tabstop>
  <tabstop>txtDatabase</tabstop>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>btnTestConnection</tabstop>
  <tabstop>btnNext</tabstop>
  <tabstop>btnBack</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>chkShowPassword</tabstop>
  <tabstop>lstTestConnection</tabstop>
  <tabstop>cmbClientTable</tabstop>
  <tabstop>cmbResultTable</tabstop>
  <tabstop>btnRefreshTables</tabstop>
  <tabstop>btnVerifyTables</tabstop>
  <tabstop>btnCreateClients</tabstop>
  <tabstop>btnCreateResults</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
